src/Sylius/Bundle/AdminBundle/EventListener/ResourceDeleteExceptionListener.php

```diff
@@ -37,12 +37,16 @@ public function onResourceDeleteException(ExceptionEvent $event): void
             return;
         }
 
+        $eventRequest = $event->getRequest();
```

Comment:
If I see correctly, there is no unit test for this class, it would be nice to add it with this case also covered